### PR TITLE
BACKLOG-15859 Make sure fieldset itself has a section assigned to it

### PR DESCRIPTION
**Pull request by AKarmanov**
## JIRA

https://jira.jahia.org/browse/BACKLOG-15859

## Description

Make sure fieldset itself has a section assigned to it

### CODE REVIEW

**Review by gflores-jahia — COMMENTED** [review]
This looks good to me, but @dgriffon let me know if you can quickly take a look as well to see if there's anything we missed.

File: src/main/java/org/jahia/modules/contenteditor/api/forms/impl/EditorFormServiceImpl.java [hunk 1]
```diff
@@ -451,6 +449,11 @@ private EditorFormFieldSet generateEditorFormFieldSet(Set<String> processedPrope
             !fieldSetEditable,
             editorFormFields
         );
+
+        // Set correct target for fieldset itself which will be used in case the fieldset doesn't have fields i. e. mixin definition without properties
+        fieldset.setTarget(new EditorFormFieldTarget(nodeType.getItemsType(), nodeType.getName(), -0.1));
```
Comment (XiangYuHe) [inline src/main/java/org/jahia/modules/contenteditor/api/forms/impl/EditorFormServiceImpl.java:454]: Curious what does the -0.1 do?
But being said does that mean that target section never worked? (As it was not assigned?)

Comment (AKarmanov) [reply src/main/java/org/jahia/modules/contenteditor/api/forms/impl/EditorFormServiceImpl.java:454]: it sets rank. It worked as long as you had at least one property defined on your definition.

**Review by dgriffon — APPROVED** [review]
Looks good to me too. Thank you!

**Pull request merged**